tools/project_manager/projectmgr.php

```diff
@@ -16,16 +16,8 @@
 require_login();
 
 $user = User::load_current();
-switch ($user->i_pmdefault) {
-    case 0:
-        $default_view = "user_all";
-        break;
-    case 1:
-        $default_view = "user_active";
-        break;
-    default:
-        $default_view = "blank";
-}
+$user_settings = Settings::get_Settings($user->username);
+$default_view = $user_settings->get_value("pm_view", "user_all");
```

Comment:
does this change the default from "blank" to "user_all"? Not that I expect the db value to be unset or outside the range of [0, 1, 2]

Reply:
It might seem like it changes the default, but the fall-through was never actually used because the value is always set by a `DEFAULT` value in the database:
```
`i_pmdefault` smallint(1) NOT NULL default '0',
```

And `0` is `user_all` which is the new default if no value is set (such as a new PM that hasn't specified a value in their preferences).

Reply:
Makes perfect sense.